platforms/android/library-compose/src/main/java/io/element/android/wysiwyg/compose/internal/RichTextEditorStyleExt.kt

```diff
@@ -89,13 +91,23 @@ internal fun TextStyle.rememberTypeface(): State<Typeface> {
 
 internal fun TextView.applyStyleInCompose(style: RichTextEditorStyle) {
     setTextColor(style.text.color.toArgb())
-    val lineHeightInPx = TypedValue.applyDimension(
-        TypedValue.COMPLEX_UNIT_DIP,
-        style.text.lineHeight.value,
-        context.resources.displayMetrics
-    )
-    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.P) {
-        lineHeight = lineHeightInPx.roundToInt()
+    if (style.text.lineHeight.isSpecified && style.text.lineHeight.value > 0f) {
```

Comment:
Could the style class enforce this requirement somehow? And provide a default line height also?

Reply:
The default line hight is already provided in `textStyle()`. But I don't think `TextUnit` actually enforces a positive value at all, and I don't think we can enforce them with annotations either. This is just here to protect agains crashes because some invalid value was passed. We could maybe log an error instead?

Also, passing `TextUnit.Unspecified` can be useful to default to the line height of the component instead of the default one contained in the `TextStyle` data class.